Silicon/TigerlakePkg/Library/TxtInit/TxtDxeLib.c

```diff
@@ -0,0 +1,867 @@
+/** @file
```

Comment:
We have to check for any restrictions with upstream. This code is from Intel folder.